products.json

```
@@ -63,6 +63,10 @@
     "name": "Console",
     "url": "https://console.cloud.google.com/home/dashboard?project={{.ProjectID}}"
   },
+  {
+    "name": "Container Registry",
+    "url": "https://console.cloud.google.com/gcr/images/{{.ProjectID}}?project={{.ProjectID}}"
+  }
```

```suggestion
  },
```

There seems to be a missing comma between the json objects

---

Thanks! Sorry for the typo